src/main/java/org/mitre/inferno/App.java

```diff
@@ -31,7 +31,13 @@ public static void main(String[] args) {
    * Only used for getting the FHIR artifacts cached.
    */
   private static void initializeValidator() {
-    new Validator("./igs/package");
```

Comment:
I'm a little concerned about changing this because [we've previously volume mounted in profiles](https://github.com/onc-healthit/inferno-site-overlay/blob/saner_config/docker-compose.yml#L40).

If a user is working with this validator and wants to load their profiles (and they are not on packages.fhir.org), how are they supposed to do it?

I'd prefer to do it outside the constructor anyways, but I don't want to throw away the ability to load specific IGs on startup.

Reply:
Ok I just reverted it